packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -97,6 +97,12 @@ The client-based rate limit framework can operate in one of three modes:
 | **Log per client**                          | The rate limit is based on the [org-wide rate limit](/docs/reference/rate-limits/) values, but the client-specific rate limit violation information is logged as System Log events. |
 | **Do nothing**                                | Rate limits aren't enforced at the client-specific level. The rate limit is based on the [org-wide rate limit](/docs/reference/rate-limits/) values. No new or additional System Log events are produced from this feature in this mode. |
 
+#### Per client rate limit
+
+As of October 12, 2021, the client-based rate limits for the OAuth 2.0 `/authorize` and `/login/login.htm` endpoints are set to **Log per client** mode. This means that if an org's client-based rate limit was previously set to **Do nothing**, the setting is changed to **Log per client** mode.
```

Comment:
Do we want to say "..., the default client-based rate limits for... "?

Reply:
👍